### ISSUE TITLE
not full backup

### ISSUE THREAD

**Issue by James99R** [ZombieStriker/ServerRestorer#24]
Plugin is just backuping everything in root folder except folder plugins world ect..

**Comment by leviem1** [comment]
Yep, can confirm on 1.15.2. This compression algorithm seems a lot like zip with extra removal steps. 😆

**Comment by leviem1** [comment]
Just took a peek at the spigot page. Appears there's not much involvement with the community, but it appears this was an issue affecting linux hosts and should be resolved in the latest update? Testing myself now.
[ServerRestorer - Fix for linux machines](https://www.spigotmc.org/resources/serverrestorer.48853/update?update=346477)
